packages/terra-core-docs/package.json

```diff
@@ -38,60 +38,60 @@
     "@cerner/terra-docs": "^1.9.0",
     "moment": "^2.29.4",
     "terra-action-footer": "^2.65.0",
-    "terra-action-header": "^2.84.0",
-    "terra-alert": "^4.83.0",
-    "terra-arrange": "^3.53.0",
+    "terra-action-header": "^2.85.0",
+    "terra-alert": "^4.84.0",
+    "terra-arrange": "^3.54.0",
     "terra-avatar": "^3.17.0",
     "terra-badge": "^3.58.0",
     "terra-base": "^5.50.0",
     "terra-breakpoints": "^2.38.0",
-    "terra-button": "^3.68.2",
-    "terra-button-group": "^3.70.0",
+    "terra-button": "^3.69.0",
+    "terra-button-group": "^3.71.0",
     "terra-card": "^3.45.0",
     "terra-cell-grid": "^1.13.0",
     "terra-content-container": "^3.39.1",
     "terra-demographics-banner": "^3.63.1",
-    "terra-dialog": "^2.81.0",
+    "terra-dialog": "^2.82.0",
     "terra-divider": "^3.38.0",
     "terra-dropdown-button": "^1.36.0",
     "terra-dynamic-grid": "^3.34.0",
     "terra-form-checkbox": "^4.21.0",
     "terra-form-field": "^4.30.0",
     "terra-form-fieldset": "^2.77.0",
     "terra-form-input": "^4.28.0",
-    "terra-form-radio": "^4.43.0",
-    "terra-form-select": "^6.48.0",
+    "terra-form-radio": "^4.44.0",
+    "terra-form-select": "^6.49.0",
     "terra-form-textarea": "^5.29.0",
     "terra-grid": "^6.34.0",
-    "terra-heading": "^4.52.0",
+    "terra-heading": "^4.53.0",
     "terra-html-table": "^1.16.0",
     "terra-hyperlink": "^2.64.0",
     "terra-i18n": "^4.40.0",
     "terra-icon": "^3.58.0",
     "terra-image": "^3.38.0",
-    "terra-list": "^4.65.0",
+    "terra-list": "^4.66.0",
     "terra-overlay": "^3.79.0",
     "terra-paginator": "^2.86.0",
     "terra-profile-image": "^3.43.0",
     "terra-progress-bar": "^4.34.1",
     "terra-responsive-element": "^5.37.0",
     "terra-scroll": "^2.35.0",
-    "terra-search-field": "^3.94.0",
-    "terra-section-header": "^2.62.0",
+    "terra-search-field": "^3.95.0",
+    "terra-section-header": "^2.63.0",
     "terra-show-hide": "^2.66.0",
     "terra-signature": "^2.39.0",
     "terra-slide-panel": "^3.46.0",
     "terra-spacer": "^3.61.0",
-    "terra-status": "^4.60.0",
-    "terra-status-view": "^4.66.0",
+    "terra-status": "^4.61.0",
+    "terra-status-view": "^5.0.0",
```

Comment:
This needs to be corrected to the `^4.67.0`

Reply:
missed while updating the version !! updated https://github.com/cerner/terra-core/pull/3948/commits/0ee1d0aec8e2e2e6f2c3bb51e1c9242218a7a507